python_tests/test_python_frontend.py

```diff
@@ -2408,6 +2408,27 @@ def fusion_func(fd: FusionDefinition, inps, matmul_fn) -> None:
                 fc = FusionCache.get()
                 fc.reset()
 
+    @unittest.skipUnless(
```

Comment:
ATen evaluation should be able to work with all architectures. Do we need this then?

Reply:
I was in two minds about putting this here. Eventually we'll need this (once it goes through Fuser). I am happy remove it for now.

Reply:
I prefer removing it for now, so that any errors on other architectures are captured properly when using ATen evaluation.